templates/web/base/admin/navigation.html

```diff
@@ -2,7 +2,7 @@
     <p>
         <strong>[% tprintf(loc('%s admin:', "%s here is the site name"), site_name) %]</strong>
       [%- FOREACH link IN allowed_links %]
-        <a href="[% c.uri_for( link ) %]">[% allowed_pages.$link.0 %]</a>
+        <a href="[% c.uri_for( '/admin/' _ link ) %]">[% allowed_pages.$link.0 %]</a>
```

Comment:
This is fine :) uri_for makes it a working relative link. However, the response priority body lists (and two template lists too) have hard-coded "/admin…" which won't work if the admin is proxied.